pkg/tuned/controller.go

```diff
@@ -445,9 +465,75 @@ func providerSync(provider string) (bool, error) {
 	return true, providerExtract(provider)
 }
 
+// switchTunedHome changes "native" container's home directory as defined by the
+// Containerfile to the container's home directory on the host itself.
+func switchTunedHome() error {
+	const (
+		ocpTunedHomeHost = "/host" + ocpTunedHome
+	)
+
+	// Create the container's home directory on the host.
+	if err := os.MkdirAll(ocpTunedHomeHost, os.ModePerm); err != nil {
+		return fmt.Errorf("failed to create directory %q: %v", ocpTunedHomeHost, err)
+	}
+
+	// Delete the container's home directory.
+	if err := util.Delete(ocpTunedHome); err != nil {
+		return fmt.Errorf("failed to delete: %q: %v", ocpTunedHome, err)
+	}
+
+	if err := util.Symlink(ocpTunedHomeHost, ocpTunedHome); err != nil {
+		return fmt.Errorf("failed to link %q -> %q: %v", ocpTunedHome, ocpTunedHomeHost, err)
+	}
+
+	err := os.Chdir(ocpTunedHome)
```

Comment:
nit: `return os.Chdir(ocpTunedHome)`?

Reply:
Absolutely, the current code is ugly. Please feel free to change in your followup PR. If not, adding to my TODO list as a cleanup.